PreMailer.Net/PreMailer.Net/PreMailer.Net.csproj

```diff
@@ -36,9 +36,8 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="AngleSharp, Version=0.9.5.41771, Culture=neutral, PublicKeyToken=e83494dcdc6d31ea, processorArchitecture=MSIL">
-      <HintPath>..\packages\AngleSharp.0.9.5\lib\net45\AngleSharp.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="AngleSharp">
+      <HintPath>..\..\..\CapitalID_AngleSharp\AngleSharp.Legacy\bin\Release\AngleSharp.dll</HintPath>

```

Comment:
Hey @driekus77 I noticed AngleSharp 0.9.6 is out now.

I just wonder how the Nu Get package reference was left in this PR? I'm not sure it would work outside of your branch, since the `HintPath` resolves a path outside of the project root?